src/clnt_vc.c

```diff
@@ -312,7 +312,7 @@ clnt_vc_ncreate2(int fd,	/* open file descriptor */
 		if (ct->ct_addr.len)
 			mem_free(ct->ct_addr.buf, ct->ct_addr.len);
 
-		if (xd->refcnt == 1) {
```

Comment:
Are you checking the NTIRPC git or the ganesha submodle ?
The change wi69676be203754bc146c6c18a6cde7fd4489825ed
is adding
@ -349,10 +349,6 @@ rpc_dplx_unref(struct rpc_dplx_rec *rec, u_int flags)
 
        if (rec->refcnt == 0) {
                t = rbtx_partition_of_scalar(&rpc_dplx_rec_set.xt, rec->fd_k);
-               if (rec->hdl.xd) {
-                       rec->hdl.xd->refcnt--;
-                       rec->hdl.xd->rec = NULL;
-               }
                REC_UNLOCK(rec);
                rwlock_wrlock(&t->lock);
                nv = opr_rbtree_lookup(&t->t, &rec->node_k);
which is, therefore, decrementing the xd refcount becuase rec just returned it's reference for xd.